src/main/java/transitionJsp/Change.java

```diff
@@ -0,0 +1,17 @@
+package transitionJsp;
+
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+
+@WebServlet("/change")
+public class Change extends HttpServlet {
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {

```

Comment:
Why do you open page with doPost method, not with doGet?
And why do you have pairs of servlets?